run3/flow/get_vn_vs_mass.py

```diff
@@ -353,6 +359,10 @@ def get_vn_vs_mass(fitConfigFileName, centClass, inFileName,
             vnFitter[iPt].SetInitialGaussianSigma(fitConfig['Sigma'][iPt], 1)
             if fixSigma[iPt]:
                 vnFitter[iPt].FixSigmaFromMassFit()
+            # nSigma4SB
+            if 'NSigma4SB' in fitConfig:
+                vnFitter[iPt].SetNSigmaForVnSB(fitConfig['NSigma4SB'][iPt])
+                print(f'NSigma4SB = {fitConfig["NSigma4SB"][iPt]}')
```

Comment:
Hi @Libeiyu ! Thanks a lot for the work. I would remove the print here though

Reply:
Hi @stefanopolitano! I am very sorry for the late reply. Is there any update on this matter? If there are no other questions, I'd like to ask for approval of this PR, please.